File: Level1/20.이상한문자만들기/20.이상한문자만들기.py
def solution(s):
    answer = ''
    oddOrEven = 0
    for i in range(len(s)):
        if s[i]==' ':
            answer += ' '
            oddOrEven = 0
        elif oddOrEven%2==0:
            answer += s[i].upper()
            oddOrEven +=1
        else:
            answer += s[i].lower()
            oddOrEven +=1
    return answer

File: Level1/20.이상한문자만들기/20.이상한문자만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    int oddOrEven = 0;
    for(int i = 0 ; i<s.size() ; i++){
        if(s[i]==' '){
            answer+=' ';
            oddOrEven=0;
        }else if(oddOrEven%2==0){
            answer+=toupper(s[i]);
            oddOrEven++;
        }else{
            answer+=tolower(s[i]);
            oddOrEven++;
        }
    }
    return answer;
}